App/App_macOS.swift

```diff
@@ -110,7 +110,7 @@ struct RootView: View {
     private let tabCloses = NotificationCenter.default.publisher(for: NSWindow.willCloseNotification)
 
     var body: some View {
-        NavigationView {
+        NavigationSplitView {
```

Comment:
In accordance to Apple's recommendation from: 
https://developer.apple.com/documentation/swiftui/migrating-to-new-navigation-types